src/components/interfaces/HMI_API.xml

```diff
@@ -503,6 +503,45 @@
     <element name="turnText"/>
 </enum>
 
+<enum name="TextFieldType">
```

Comment:
I did notice the renaming proposal as well, but I was not sure whether we should proceed with the renaming on this feature branch or create a follow-up branch.
I went ahead and performed the proposed renamings on the latest commit.